daemons/gptp/common/ieee1588.hpp

```diff
@@ -499,7 +501,8 @@ class HWTimestamper {
 	 * @param  frequency_offset Frequency offset
 	 * @return false
 	 */
-	virtual bool HWTimestamper_adjclockrate( float frequency_offset )
+	virtual bool HWTimestamper_adjclockrate
```

Comment:
Thanks for the feedback. I will check into those function descriptions. I tried to transpose all of those.

It may look funny, but const is fine for *adjclockrate because while it changes the state of the driver or device, it doesn't - currently at least - change the state of the timestamper object.

As I said below, I think using const is good practice.